bin/importPdoMetadata.php

```diff
@@ -0,0 +1,25 @@
+<?php
+$baseDir = dirname(dirname(__FILE__));
+
+require_once $baseDir . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . '_autoload.php';
+require_once $baseDir . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'config.php';
+
+# Iterate through configured metadata sources and ensure
+# that a PDO source exists.
+foreach ($config['metadata.sources'] as $s) {
+	# If pdo is configured, create the new handler and add in the metadata sets.
+	if ("pdo" === $s['type']) {
+		$mdshp = new SimpleSAML_Metadata_MetaDataStorageHandlerPdo($s);
+		$mdshp->initDatabase();
+		foreach (glob("metadata/*.php") as $filename) {
```

Comment:
I'm assuming this allows you to periodically import metadata into the DB, right? In that case, we might be assuming too much here if we expect all metadata to be in `metadata/{set}.php`, as that could change depending on the source. For instance, it could make sense to use this to pull metadata generated by metarefresh.